docs/configuration.rst

```diff
@@ -21,6 +21,12 @@ The complete list of command line options is:
   ``--memray-bin-prefix``
     Prefix to use for the binary dump (by default a random UUID4 hex)
 
+  ``--stacks=STACKS`` 
+    Show the N stack entries when showing tracebacks of memory allocations
+
+  ``--native``
+    Show native frames when showing tracebacks of memory allocations (will be slower)
```

Comment:
Let's say
```suggestion
    Include native frames when showing tracebacks of memory allocations (will be slower)
```
instead. That's a minor tweak, but it makes it clear that the native frames are shown _in addition_ to the Python frames, rather than instead of.